lamindb/dev/_run_context.py

```diff
@@ -468,6 +468,26 @@ def _track_notebook(
             save_or_load(f"saved: {transform}")
         else:
             # check whether there was an update
+            if (
+                transform.source_file_id is not None
+                or transform.latest_report_id is not None
+            ):
+                response = input(
+                    "You already saved a source file and a report for this transform"
+                    " version! Do you want to increase the version number? (y/n)"
+                )
+                if response == "y":
+                    if _env in ("lab", "notebook") and is_interactive:
```

Comment:
Hey @Koncopd, do we need `_env in ("lab", "notebook")`?

The only thing that matters should be `is_interactive`, right?

Reply:
This is potentially different. Also this is used before `is_interactive` is set. I would really avoid changing something in the logic here because we don't really test the interactive features, it is pretty easy to break something subtle without realizing. It need a refactor, but very careful and not as a part of a PR with another purpose.

Reply:
At this exact place it looks you are right, but see above...

Reply:
> It needs a refactor, but very careful and not as a part of a PR with another purpose.
Agreed!